src/main.cpp

```diff
@@ -83,7 +84,14 @@ void run_inverter()
 	while (true)
 	{
 		mutex_enter_blocking(&lcmMutex);
-		float frequency = calculate_frequency(lcm.velocity, lcm.throttle);
+
+		// Calculate time difference in microseconds and check if the difference is
+		// greater than the expiration time. If so, set frequency to 0. Otherwise,
+		// set frequency to the data received.
```

Comment:
Nitpick [non-blocking]: comment seems a bit too verbose, describing the code literally rather than explaining intent.